### DISCUSSION TITLE
Local adaptive quantile regression splitting

### DISCUSSION THREAD

**Discussion by mcberko** [kogalur/randomForestSRC#440] [Q&A]
Hi there,

I was reading the documentation on how your package implements QRFs and came across "local adaptive quantile regression splitting". Do you have more details on how this works? I couldn't find any papers with this term. Is this related to conformal prediction methods?

Thanks!
Matt

**Comment by ishwaran** [comment]
It's a gradient based approach.  The derivative of the loss function at the cell estimator is used for the Y-outcomes which leads to a splitting rule that is similar to gini splitting but where Y are class labels determined by quantiles in the node to be split.  The locally adaptive splitting rule is an improvement on this.  We don't have a vignette describing this (as yet).  However, it appears in my text book to be published this year.

**Reply by mcberko** [reply]
Thanks! Did you implement this as the default splitting rule (as opposed to, say, the usual squared error loss than Meinshausen uses) because of some simulation results you obtained, or some other reason?

**Reply by ishwaran** [reply]
All of this is explained in the help file.  The default method used is not the Meinhausen method and the default splitting rule is not MSE but adaptive quantile splitting.   Both of those methods can be invoked if you want by setting options `method` and `splitrule`.  We opted against the Meinhausen method because of the mounting evidence that it yields biased estimates of the prediction interval.  We chose quantile splitting over MSE because the pinball loss is the proper scoring function in quantile estimation.

**Reply by mcberko** [reply]
Thanks again! By any chance, is the splitting criterion you added functionality for similar to (or the same as) the splitting criterion defined in Athey et al. (2019) where they consider QRFs as an application of their framework?

**Reply by ishwaran** [reply]
It's possible there are some similarities in the splitting criterion.  I should mention that the splitting rule (and the tree) is designed to produce simultaneous quantile values - by default 100 values are used.  Therefore, the splitting rule uses 100 latent variables which arise from the gradient approach mentioned above.  I don't think this is similar to Athey (2019).  Also, I don't think the splitting rule is the only important aspect to out-performing "traditional" QRF.  We also use the OOB residual method described in Zhang (2019) for construction of the prediction intervals.
